index.html

```diff
@@ -2499,6 +2524,14 @@ <h3>
             A user agent can present this icon where space constraints and/or
             color requirements differ from those of the application icon.
           </dd>
+          <dt>
+            <dfn data-lt="badge maskable">maskable</dfn>:
+          </dt>
+          <dd>
+            The image is designed with <a href="#icon-masks">maskable</a> in mind,
```

Comment:
If you want to link to the "Icon masks and safe zone" section and not to the `maskable` definition, it'd be better to expand `<a href="#icon-masks">icon masks and safe zone</a>`.

(`<a>markable</a>` should work if you link to its definition.)